tests/scripts/all.sh

```diff
@@ -2440,6 +2444,10 @@ component_test_psa_crypto_config_accel_pake_no_fallback () {
     scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_ALG_JPAKE
     scripts/config.py unset MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED
 
+    # Dynamic secure element support is a deprecated feature and needs to be disabled here.
```

Comment:
If this change is accepted I will have to add it also to at least two other `accel` components (failing on CI).